docs/source/writing-tests.md

```diff
@@ -103,7 +103,7 @@ is equivalent to
 var=$(command args ...)
 ```
 
-#### Comment syntax
+## Comment syntax
```

Comment:
This lost two levels but still looks good. Just asking because you specifically said **all** were one level off.

Reply:
Yes, the two-level thing was a very late change as I re-viewed the patch. Didn't want to change the commit message over this minutia.